streams/src/main/java/org/apache/kafka/streams/kstream/GlobalKTable.java

```diff
@@ -67,4 +67,10 @@
  */
 @InterfaceStability.Evolving
 public interface GlobalKTable<K, V> {
+    /**
+     * Get the name of the local state store used that can be used to query this {@code KTable}.
```

Comment:
typo: `used that can be used` 

Is this a c&p error? if yes, please fix where you copied it from, too.